plugins-slim.yml

```diff
@@ -0,0 +1,2 @@
+- github.com/prometheus/prometheus/discovery/consul
+- github.com/prometheus/prometheus/discovery/kubernetes
```

Comment:
Let the bike-shedding commence.

Reply:
💯 

Would a better name for this "distribution" of the Prometheus binary be something like "prometheus-kubernetes"?

Reply:
since it also includes consul .. no .. but this is just a proposal. I am happy to close it as is if you think it's not necessary.